src/resources/Resolver.ts

```diff
@@ -47,6 +50,16 @@ export class Resolver {
       if (responseMappingTemplate) {
         Properties.ResponseMappingTemplate = responseMappingTemplate;
       }
+    } else {
+      if (this.config.code) {
+        Properties.Code = this.resolveJsCode(this.config.code);
+      } else if (!this.config.code) {
+        Properties.Code = DEFAULT_JS_RESOLVERS;
```

Comment:
This default should only be used for PIPELINE resolvers. 
For UNIT, if neither `request`/`response` nor `code` are present, we should throw an error.

Ideally, we'd want to do that in [validation](https://github.com/sid88in/serverless-appsync-plugin/blob/master/src/validation.ts#L277-L279)
The rule might be a bit complicated.
We probably can do that with an if tule like [here](https://github.com/sid88in/serverless-appsync-plugin/blob/master/src/validation.ts#L80)

Reply:
`request`, `response`, `code`  [are not necessary for UNIT resolver with direct lambda(datasource)](https://github.com/sid88in/serverless-appsync-plugin/blob/227d479b41ec9b44fc5c1772272d2328395b8ee1/src/__tests__/resolvers.test.ts#L79).
So we cannot add that validation.

Instead, I modified [doc/resolvers.md](https://github.com/sid88in/serverless-appsync-plugin/pull/613/files#diff-38627408e5827a4ac45e70b468a33f7042d8e251c93e774ddfc3769c31ffbd80).

Reply:
Thanks. 
You're right. I forgot about direct Lambda.

In any case, this still means that if `code` is not present, for `UNIT`, we should NOT use the default. 
Especially if we're using direct Lambda.

Reply:
I would just remove this condition

`if(isUnitJsResolver || isPipelineJsResolver)`

and add 

`} else if (this.config.kind === 'PIPELINE' && !isVTLResolver) {`